src/scripts/main.js

```diff
@@ -1,3 +1,29 @@
 'use strict';
 
-// write your code here
+const logo = document.querySelector('.logo');
+
+const addMessage = (messageText, className) => {
+  const message = document.createElement('div');
+
+  message.textContent = messageText;
+  message.className = className;
+  document.body.append(message);
+};
+
+// eslint-disable-next-line no-unused-vars
+const firstPromise = new Promise(resolve => {
+  logo.addEventListener('click', () => {
+    const messageText = 'Promise was resolved!';
+
+    addMessage(messageText, 'message');
+  });
+});
+// eslint-disable-next-line no-unused-vars
+const secondPromise = new Promise((resolve, reject) => {
+  const errorMessageText = 'Promise was rejected!';
```

Comment:
необязательно записывать это в переменные, тогда и линтер отключать не нужно будет
```suggestion

new Promise(resolve => {
  logo.addEventListener('click', () => {
    const messageText = 'Promise was resolved!';

    addMessage(messageText, 'message');
  });
});

new Promise((resolve, reject) => {
  const errorMessageText = 'Promise was rejected!';
```

Reply:
Тобто в інших роботах як ТИ апруфнув, виключений лінтер тебе не смутив. Окей, поняв.